app/frontend/src/assets/languages/locale-en.json

```diff
@@ -102,6 +102,7 @@
   "sitnet_examcode": "Code",
   "sitnet_examcode_and_name": "Exam code and name",
   "sitnet_examination_location": "Exam room",
+  "sitnet_examination_location_and_machine":"Exam room and computer",
```

Comment:
Näihin käännöksiin kantsii aina laittaa vaikka "Tenttitila ja -kone EN/SV" niin hoksaa, että tuo vaatii käännöksen asiakkaalta, kun me devaajathan emme näitä käännöksiä tee :)